backend/shop/tests/test_views.py

```diff
@@ -487,3 +490,118 @@ def test_get_shop_profiles_success(self):
 
         # Assert that the returned groups match the expected groups
         self.assertEqual(set(returned_groups), set(expected_groups))
+
+
+class TestProductGet(APITestCase):
+    def setUp(self):
+        self.user = CoreUserFactory()
+        self.url_all = reverse("product_get_all")
```

Comment:
It will be cleaner for whole test class if you create two users:
- normal user (that has view permission)
- editor user (that have change permission)